cime_config/machines/config_machines.xml

```diff
@@ -3430,6 +3430,8 @@
     <MAX_TASKS_PER_NODE compiler="pgigpu">18</MAX_TASKS_PER_NODE>
     <MAX_TASKS_PER_NODE compiler="gnugpu">42</MAX_TASKS_PER_NODE>
     <MAX_TASKS_PER_NODE compiler="ibmgpu">42</MAX_TASKS_PER_NODE>
+	<!-- Need to activate following attribute after CIME update from upstream -->
+    <!-- <MAX_GPUS_PER_NODE>6</MAX_GPUS_PER_NODE> -->
```

Comment:
Not sure we need this var? CIME might need an additional python check to prevent writes like
```
ERROR: No variable MAX_GPUS_PER_NODE found in case
```
rather than defining the XML var here to avoid the write.

Reply:
Related discussion at https://github.com/ESMCI/cime/issues/4258
I do want preview_run output to be correct. Right now, it's not and causes confusion for those who don't know about this issue.

Reply:
Ok. Meanwhile, can we get this PR done?
I would prefer to leave that comment in to revisit later.